milestone-3/frontend/src/components/home.tsx

```diff
@@ -0,0 +1,31 @@
+
+import "./home.css";
+import RecipePreview from "./recipePreview";
+import recipes from "../recipeData.json";
+import {useState, useEffect} from 'react';
+import {Recipe} from '../types';
+
+export default function Home() {
+  const [externalRecipes, setExternalRecipes] = useState<Recipe[]>([]);
+  useEffect(() => {
+    fetch("https://bootcamp-milestone-4.onrender.com/recipe")
+      .then((res) => res.json())
+      .then((data) => setExternalRecipes(data));
+  }, []);
```

Comment:
This should be either taken out or moved to app.tsx. If you grab the external recipe's here they will all show up in the previews on the home.tsx page but if you click on them they will lead to a blank page. This is because you aren't generating a route for there external recipes in your app.tsx and passing along the data through props to each of those RecipePage components.